EPI/parse-GTRI.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+if __name__ == '__main__':
+    gtri_json_filename = "models/CHIME-SIR-GTRI.json"
```

Comment:
Maybe we want to pass the input file name as argument? Just in case we want to parse other potential models from GTRI that are not just CHIME. 
`   main(sys.argv[1])`

Reply:
Sure